src/chains/goerli.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "goerli",
-  "chainId": 6284,
-  "networkId": 6284,
+  "chainId": 5,
+  "networkId": 5,
   "comment": "Cross-client PoA test network",
   "url": "https://github.com/goerli/testnet",
   "genesis": {
-    "hash": "0xfa57319d09fd8a32faaf18d338c8a925a5a7975285bf29ecd024e083cba8abb1",
-    "timestamp": "0x5bdda800",
+    "hash": "0xbf7e331f7f7c1dd2e05159666b3bf8bc7a8a3a9eb1d518969eab529dd9b88c1a",
+    "timestamp": "0x5c51a607",
     "gasLimit": 10485760,
     "difficulty": 1,
     "nonce": "0x0000000000000000",
-    "extraData": "0x2249276d20646f6e652077616974696e672e2e2e20666f7220626c6f636b2066696e616c69747922202d2049676779270000000001fa1804c408085d9c57eeb167ce953c99b6cb1e20794Fd02933F303FbA550bd1fe2f0649E3576eB0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-    "stateRoot": "0x0f410b6a6eae3d3156eccd966ac842a4c545c47921b9fe36386de18152cfddcf"
+    "extraData": "0x22466c6578692069732061207468696e6722202d204166726900000000000000e0a2bd4258d2768837baa26a28fe71dc079f84c70000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    "stateRoot": "0x5d6cded585e73c4e322c30c2f782a336316f17dd85a4863b9d838d2d4b8b3008"
```

Comment:
Looks all good! Just for my information, did you compute `stateRoot` manually? couldn't find it in the goerli repo.

Reply:
See https://twitter.com/HolgerD77/status/1092735223274577920 😄.